### ISSUE TITLE
Youtube Chapters Support

### ISSUE THREAD

**Issue by CasperMcFadden95** [anxdpanic/plugin.video.youtube#852]
### What Problem Does This Solve? 

YouTube videos can have chapters, it would be nice if these were visible in the Kodi player.

An example video is https://www.youtube.com/watch?v=b1Fo_M_tj6w

Thanks

**Comment by MoojMidge** [comment]
There is no way, that I know of, for a video plugin to add chapter markers for a video stream.

It would be relatively simple if an EDL files could be loaded, but Kodi will not do this for an internet stream.

The only possible way that I can think to do something similar would be to create a custom video window that loads a window property as the info of a ranges control, with the plugin writing the chapter markers to the window property.

Not sure if that would even work, and don't have the time to try.

**Comment by smurfy** [comment]
I have no idea reagarding whats possible and whats not via plugin/addon, but it seems as an user i can create bookmarks for a YT video, couldnt the video plugin just attach those?

**Comment by MoojMidge** [comment]
> as an user i can create bookmarks for a YT video, couldnt the video plugin just attach those?

No, there is no way for a plugin to create these bookmarks automatically
